examples/wide_deep/run.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+export PYTHON_ROOT=./Python
+export LD_LIBRARY_PATH=${PATH}
+export PYSPARK_PYTHON=${PYTHON_ROOT}/bin/python
+export SPARK_YARN_USER_ENV="PYSPARK_PYTHON=Python/bin/python"
+export PATH=${PYTHON_ROOT}/bin/:$PATH
+export HADOOP_HOME=/opt/cloudera/parcels/CDH-5.11.0-1.cdh5.11.0.p0.34
```

Comment:
Does this depend on something particular in Cloudera Hadoop distro or should this work with any?